bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/binding/BaseThingHandler.java

```diff
@@ -96,6 +101,22 @@ public void removedService(final ServiceReference reference, final Object servic
             }
         };
         thingRegistryServiceTracker.open();
+        linkRegistryServiceTracker = new ServiceTracker(this.bundleContext, ItemChannelLinkRegistry.class.getName(),
```

Comment:
Note that this is not related to this PR, I didn't change anything in this respect.

The reason is that handlers are created by ThingFactories - we have a handler for each thing and not just one per binding. And afair, although there are also factories for DS, this didn't really match here.